docs/quickstart.rst

```diff
@@ -0,0 +1,190 @@
+    Zarr format        : 3
+    Data type          : DataType.float32
+    Shape              : (100, 100)
+    Chunk shape        : (10, 10)
+    Order              : C
+    Read-only          : False
+    Store type         : LocalStore
+    Codecs             : [{'endian': <Endian.little: 'little'>}, {'level': 0, 'checksum': False}]
+    No. bytes          : 40000 (39.1K)
+
+This compresses the data using the Zstandard codec with shuffle enabled for better compression.
+
+Hierarchical Groups
+-------------------
+
+Zarr allows you to create hierarchical groups, similar to directories::
+
+    >>> # Create nested groups and add arrays
+    >>> root = zarr.group("data/example-2.zarr")
+    >>> foo = root.create_group(name="foo")
+    >>> bar = root.create_array(
+    ...     name="bar", shape=(100, 10), chunks=(10, 10)
+    ... )
+    >>> spam = foo.create_array(name="spam", shape=(10,), dtype="i4")
+    >>>
+    >>> # Assign values
+    >>> bar[:, :] = np.random.random((100, 10))
+    >>> spam[:] = np.arange(10)
+    >>>
+    >>> # print the hierarchy
+    >>> root.tree()
+    /
+    └── foo
+        └── spam (10,) int32
+    <BLANKLINE>
+
+This creates a group with two datasets: ``foo`` and ``bar``.
+
+Persistent Storage
+------------------
+
+Zarr supports persistent storage to disk or cloud-compatible backends. While examples above
+utilized a :class:`zarr.storage.LocalStore`, a number of other storage options are available,
+including the :class:`zarr.storage.ZipStore` and :class:`zarr.storage.FsspecStore`::
+
+    >>> # Store the array in a ZIP file
+    >>> store = zarr.storage.ZipStore("data/example-3.zip", mode='w')
+    >>>
+    >>> z = zarr.create_array(
+    ...     store=store,
+    ...     mode="w",
+    ...     shape=(100, 100),
+    ...     chunks=(10, 10),
+    ...     dtype="f4"
+    ... )
+    >>>
+    >>> # write to the array
+    >>> z[:, :] = np.random.random((100, 100))
+    >>>
+    >>> # the ZipStore must be explicitly closed
+    >>> store.close()
+
+To open an existing array::
+
+    >>> # Open the ZipStore in read-only mode
+    >>> store = zarr.storage.ZipStore("data/example-3.zip", read_only=True)
+    >>>
+    >>> z = zarr.open_array(store, mode='r')
+    >>>
+    >>> # read the data as a NumPy Array
+    >>> z[:]
+    array([[0.66734236, 0.15667458, 0.98720884, ..., 0.36229587, 0.67443246,
+            0.34315267],
+        [0.65787303, 0.9544212 , 0.4830079 , ..., 0.33097172, 0.60423803,
+            0.45621237],
+        [0.27632037, 0.9947008 , 0.42434934, ..., 0.94860053, 0.6226942 ,
+            0.6386924 ],
+        ...,
+        [0.12854576, 0.934397  , 0.19524333, ..., 0.11838563, 0.4967675 ,
+            0.43074256],
+        [0.82029045, 0.4671437 , 0.8090906 , ..., 0.7814118 , 0.42650765,
+            0.95929915],
+        [0.4335856 , 0.7565437 , 0.7828931 , ..., 0.48119593, 0.66220033,
+            0.6652362 ]], shape=(100, 100), dtype=float32)
+
+Cloud Storage Backends
```

Comment:
I would merge with the previous section or at least talk about it before ZipStore